protobuf/CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 ## 0.14.0
 
+* Support for proto3 json (json with field names as keys) 
+  - encoding and decoding.
+  - Support for well-known types.
+  - Use `GeneratedMessage.toProto3Json()` to encode and `GeneratedMessage.mergeFromProto3Json(json)`
```

Comment:
General question about the naming---from what I can tell, we messed up a while ago by calling the jspblite2 serializers `writeToJson` / `fromJson`.

Should this PR or a future change be a breaking change to rename these things? It seems to me that e.g. `toProto3Json` should really be `toJson`, and the current `toJson` should be `toJspbLite2`, etc.

Reply:
Good question.
I would prefer we have no toJson at all (we could start by making `toJspbLite2`   available and the deprecating toJson).
I think that explicitly choosing flavor is a good thing given the abundance of json formats.

Reply:
A `toJson` method interacts with the JSON encoder (if no `toEncodable` method is provided, the default just dynamically calls `toJson` on any unrecognzied object). If there is more than one way to JSON-ify an object, then it may be safer to not have any *default*, unless one of them is really obviously the one you almost always want.